bool data
int32 gripperTwist
int32 eeGroundRot
int32 eeAirRot
int32 gripperWidth